parsers/requirements.txt

```diff
@@ -18,3 +18,5 @@ testfixtures==6.0.0
 opencv-python==4.2.0.32
 xlrd==1.1.0
 freezegun==0.3.15
+websockets==8.1
```

Comment:
Is this (and the websockets import in the parser) still needed, or does the SignalR library take care of this?

Reply:
No, nice catch! I removed it :-)